### DISCUSSION TITLE
Custom Components like 'code_editor' in form.

### DISCUSSION THREAD

**Discussion by ahmedsakri** [manojadams/mui-forms#48] [General]
**Is your feature request related to a problem? Please describe.**
I want to add or use a custom component like a code editor by monaco-editor/react

**Describe the solution you'd like**
A way i can support this or use the code_editor display type?

**Describe alternatives you've considered**
A clear and concise description of any alternative solutions or features you've considered

**Additional context**

**Comment by manojadams** [comment]
@ahmedsakri Please check the following documentation link for custom components:
https://mui-forms.vercel.app/custom-fields/about#components
